aggregator/.gitignore

```diff
@@ -0,0 +1,40 @@
+#Maven
```

Comment:
Can we have only one `.gitignore` file at the repo root level for all Java modules?